priv/cabbage/apps/itest/test/itest/watcher_info_api_test.exs

```diff
@@ -91,6 +137,17 @@ defmodule WatcherInfoApiTest do
     Itest.Poller.pull_balance_until_amount(address, amount)
   end
 
+  defp is_all_tx_behind_timestamp(transactions, timestamp) do
+    Enum.reduce(
```

Comment:
probably `Enum.any?` would be better to read

Reply:
I was looking for something like that lol thanks

Reply:
I'm late 😅  and nitpicking and pedantic 😅 😅  How about `Enum.all?` to avoid the brain fatigue from the negation.

If we're worried of full list iteration perf, it's already taken care off by the implementation:

> Iterates over the `enumerable` and invokes `fun` on each element. When an invocation of `fun` returns a falsy value (`false` or `nil`) iteration stops immediately and `false` is returned. In all other cases `true` is returned.
>
> https://hexdocs.pm/elixir/Enum.html#all?/2

Reply:
yey all is better